src/CalcViewModel/Common/CopyPasteManager.cpp

```diff
@@ -37,11 +38,9 @@ static const wstring c_binProgrammerChars = L"[0-1]+((_|'|`)[0-1]+)*";
 static const wstring c_uIntSuffixes = L"[uU]?[lL]{0,2}";
 
 // RegEx Patterns used by various modes
-static const array<wregex, 1> standardModePatterns = { wregex(c_wspc + c_signedDecFloat + c_wspc) };
-static const array<wregex, 2> scientificModePatterns = {
-    wregex(L"(" + c_wspc + L"[-+]?)|(" + c_wspcLParenSigned + L")" + c_signedDecFloat + c_wspcRParens),
-    wregex(L"(" + c_wspc + L"[-+]?)|(" + c_wspcLParenSigned + L")" + c_signedDecFloat + L"e[+-]?\\d+" + c_wspcRParens)
-};
+static const array<wregex, 1> standardModePatterns = { wregex(c_wspc + c_signedDecFloat + c_optionalENotation + c_wspc) };
```

Comment:
>c_signedDecFloat + c_optionalENotation [](start = 71, length = 38)

Does this work?  I believe as written this only catches the alternate half of c_signedDecFloat.  ie: This will match ".5e3" but not "0.5e3".  I think you need parens around c_signedDecFloat

Reply:
Good catch! the second commit to exclude `.` (see conversations above) broke everything 🤦‍♂️
 
It's now fixed!